src/rdkafka_telemetry_encode.h

```diff
@@ -29,11 +29,135 @@
 #ifndef _RDKAFKA_RDKAFKA_TELEMETRY_ENCODE_H
 #define _RDKAFKA_RDKAFKA_TELEMETRY_ENCODE_H
 
+#include "rdtypes.h"
+
+#define TELEMETRY_METRIC_PREFIX "org.apache.kafka."
+
+#define TELEMETRY_METRIC_INFO(rk)                                              \
+        (rk->rk_type == RD_KAFKA_PRODUCER                                      \
+             ? RD_KAFKA_TELEMETRY_PRODUCER_METRICS_INFO                        \
+             : RD_KAFKA_TELEMETRY_CONSUMER_METRICS_INFO)
+
+#define TELEMETRY_METRIC_CNT(rk)                                               \
+        (rk->rk_type == RD_KAFKA_PRODUCER                                      \
+             ? RD_KAFKA_TELEMETRY_PRODUCER_METRIC__CNT                         \
+             : RD_KAFKA_TELEMETRY_CONSUMER_METRIC__CNT)
+
+
 typedef enum {
         RD_KAFKA_TELEMETRY_METRIC_TYPE_SUM,
         RD_KAFKA_TELEMETRY_METRIC_TYPE_GAUGE,
 } rd_kafka_telemetry_metric_type_t;
 
+typedef enum {
+        RD_KAFKA_TELEMETRY_METRIC_PRODUCER_CONNECTION_CREATION_RATE,
+        RD_KAFKA_TELEMETRY_METRIC_PRODUCER_CONNECTION_CREATION_TOTAL,
+        RD_KAFKA_TELEMETRY_METRIC_PRODUCER_NODE_REQUEST_LATENCY_AVG,
+        RD_KAFKA_TELEMETRY_METRIC_PRODUCER_NODE_REQUEST_LATENCY_MAX,
```

Comment:
Refactored to per broker metric in https://github.com/confluentinc/librdkafka/pull/4515